<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gains</class>
 <widget class="QDialog" name="Gains">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>713</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gains</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(220,220,220);</string>
  </property>
  <widget class="QPushButton" name="GAIN_BUT">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>17</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(60, 255, 73);
background-color: rgb(125, 153, 126);
border: 2px solid rgb(100, 25, 123);</string>
   </property>
   <property name="text">
    <string>CALCULATE</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>691</width>
     <height>471</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>172</number>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>30</number>
   </attribute>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Bought_Price</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Current_Price</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Performance</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="Overall_gain">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>630</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 128);
background-color: rgb(125, 153, 126);
border: 2px solid rgb(100, 25, 173);</string>
   </property>
   <property name="text">
    <string>Over All Gain</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Gain_disp">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>630</y>
     <width>160</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(60, 255, 73);
background-color: rgb(125, 153, 126);
border: 2px solid rgb(100, 25, 173);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>630</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>48</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
